docker-compose.dev.yml

```diff
@@ -10,11 +10,6 @@ services:
     depends_on:
       db:
         condition: service_healthy
-
-    depends_on:
```

Comment:
question: was this just a redundant clause?

Reply:
Yes. Was a blown merge somewhere.